mne/report.py

```diff
@@ -887,11 +898,18 @@ def add_images_to_section(self, fnames, captions, scale=None,
             img_klass = self._sectionvars[section]
 
             # Convert image to binary string.
-            im = Image.open(fname)
             output = BytesIO()
-            im.save(output, format='png')
+            im = Image.open(fname)
+            if image_format == 'png':
+                im.save(output, format='png')
+            elif image_format == 'gif':
+                with open(fname, 'rb') as f:
+                    output.write(f.read())
```

Comment:
Actually, I'm thinking that the idea may have been to convert all images into a png format so as to simplify the html part; in which case we could keep like that for now?